pennylane/_qubit_device.py

```diff
@@ -385,56 +385,56 @@ def analytic_probability(self, wires=None):
         """
         raise NotImplementedError
 
-    def estimate_probability(self, wires=None):
+    def estimate_probability(self, subsystems=None):
```

Comment:
my current favourite is ``wire_indices``, which is clear, but long!

Reply:
We could also keep the name ``wires`` and just do the translation on the same variable name. But this would be extremely opaque to coders. The advantage is that we would not have to refactor each and every plugin that overwrites the functions...

Reply:
I refactored everything with ``registers``.

It is not perfect, because the devices' ``register`` is an object like ``Wires=[0, 3, 'a']``, and ``registers`` in this register could also be interpreted as the wire labels. But somehow ``registers`` has a neutral connotation of ``the ith register``...